models/dex/arbitrum/dex_arbitrum_sandwiches.sql

```diff
@@ -0,0 +1,100 @@
+{{ config(
+        schema = 'dex_arbitrum',
+        alias ='sandwiches',
+        partition_by = ['block_date'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['sandwiched_pool', 'frontrun_tx_hash', 'frontrun_taker', 'frontrun_index', 'currency_address'],
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "sector",
+                                "dex",
+                                \'["hildobby"]\') }}'
+        )
+}}
+
+WITH trades AS (
+    SELECT dt.project
+    , dt.version
+    , date_trunc("day", dt.block_time) AS block_date
+    , dt.block_time
+    , t.block_number
+    , dt.token_sold_address
+    , dt.token_bought_address
+    , dt.token_sold_symbol
+    , dt.token_bought_symbol
+    , dt.taker
+    , dt.tx_hash
+    , dt.tx_from
+    , dt.project_contract_address
+    , dt.evt_index
+    , t.index
+    , MIN(t.gas_price) AS gas_price
+    , MIN((t.gas_price/POWER(10, 18))*t.gas_used) AS tx_fee
+    , SUM(COALESCE(dt.token_sold_amount_raw, 0)) AS token_sold_amount_raw
+    , SUM(COALESCE(dt.token_bought_amount_raw, 0)) AS token_bought_amount_raw
+    , SUM(COALESCE(dt.token_sold_amount, 0)) AS token_sold_amount
+    , SUM(COALESCE(dt.token_bought_amount, 0)) AS token_bought_amount
+    , SUM(COALESCE(dt.amount_usd, 0)) AS amount_usd
+    FROM {{ ref('dex_trades') }} dt
+    INNER JOIN {{ source('arbitrum', 'transactions') }} t ON t.block_time=dt.block_time
```

Comment:
similar to recent PR, can you add the incremental filters on joins to base tables? it's a good general rule of thumb for all joins to tables with event-driven or time series data (i.e. base, prices, most decoded)

Reply:
same idea for all blockchains